specs.json

```diff
@@ -1,3 +1,304 @@
+  "https://drafts.csswg.org/css-size-adjust-1/",
+  "https://drafts.csswg.org/css-sizing-4/ delta",
+  "https://drafts.csswg.org/css-transitions-2/ delta",
+  "https://drafts.csswg.org/css-tv/",
+  "https://drafts.csswg.org/scroll-animations-1/",
+  "https://drafts.fxtf.org/compositing-2/",
+  "https://drafts.fxtf.org/filter-effects-2/ delta",
+  "https://fetch.spec.whatwg.org/",
+  "https://fullscreen.spec.whatwg.org/",
+  "https://html.spec.whatwg.org/multipage/",
+  "https://immersive-web.github.io/hit-test/",
+  "https://infra.spec.whatwg.org/",
+  "https://mimesniff.spec.whatwg.org/",
+  "https://notifications.spec.whatwg.org/",
+  "https://quirks.spec.whatwg.org/",
+  "https://storage.spec.whatwg.org/",
+  "https://streams.spec.whatwg.org/",
+  "https://svgwg.org/specs/animations/",
+  "https://url.spec.whatwg.org/",
+  "https://w3c.github.io/accname/",
+  "https://w3c.github.io/contentEditable/",
+  "https://w3c.github.io/core-aam/",
+  "https://w3c.github.io/gamepad/extensions.html",
+  "https://w3c.github.io/media-playback-quality/",
+  "https://w3c.github.io/reporting/",
+  "https://w3c.github.io/ServiceWorker/",
+  "https://w3c.github.io/web-nfc/",
+  "https://w3c.github.io/webappsec-feature-policy/",
+  "https://webbluetoothcg.github.io/web-bluetooth/",
+  "https://wicg.github.io/background-fetch/",
+  "https://wicg.github.io/BackgroundSync/spec/",
+  "https://wicg.github.io/badging/",
+  "https://wicg.github.io/construct-stylesheets/",
+  "https://wicg.github.io/cookie-store/",
+  "https://wicg.github.io/cors-rfc1918/",
+  "https://wicg.github.io/CSS-Parser-API/",
+  "https://wicg.github.io/element-timing/",
+  "https://wicg.github.io/entries-api/",
+  "https://wicg.github.io/event-timing/",
+  "https://wicg.github.io/frame-timing/",
+  "https://wicg.github.io/geolocation-sensor/",
+  "https://wicg.github.io/InputDeviceCapabilities/",
+  "https://wicg.github.io/js-self-profiling/",
+  "https://wicg.github.io/keyboard-lock/",
+  "https://wicg.github.io/keyboard-map/",
+  "https://wicg.github.io/largest-contentful-paint/",
+  "https://wicg.github.io/layout-instability/",
+  "https://wicg.github.io/native-file-system/",
+  "https://wicg.github.io/netinfo/",
+  "https://wicg.github.io/origin-policy/",
+  "https://wicg.github.io/page-lifecycle/",
+  "https://wicg.github.io/permissions-request/",
+  "https://wicg.github.io/permissions-revoke/",
+  "https://wicg.github.io/priority-hints/",
+  "https://wicg.github.io/shape-detection-api/",
+  "https://wicg.github.io/speech-api/",
+  "https://wicg.github.io/video-raf/",
+  "https://wicg.github.io/visual-viewport/",
+  "https://wicg.github.io/web-locks/",
+  "https://wicg.github.io/web-share-target/",
+  "https://wicg.github.io/web-transport/",
+  "https://wicg.github.io/webusb/",
+  {
+    "url": "https://www.khronos.org/registry/webgl/specs/latest/1.0/",
+    "name": "webgl1"
+  },
+  {
+    "url": "https://www.khronos.org/registry/webgl/specs/latest/2.0/",
```

Comment:
is there any reason webgl2 is not described as a delta spec from webgl1?

(I'm not sure I understand if it even matters)

Reply:
I hadn't noticed that this was a delta spec in practice :)

It is different from other delta specs that we have, in that the "delta" status does not seem to be intented to disappear over time.

The "delta" flag impacts CSS/IDL extracts in Reffy: Reffy only produces these extracts for specs that are not flagged as "delta".

We may want to treat WebGL 1.0 and WebGL 2.0 as two separate specs in practice. If we flag the WebGL 2.0 as "delta", Reffy will only produce an IDL extract for WebGL 1.0. If we don't, Reffy will only produce an IDL for WebGL 2.0. Neither is what we need for WPT, which needs the two extracts (`webgl1.idl` and `webgl2.idl`, see https://github.com/web-platform-tests/wpt/tree/master/interfaces).

Reply:
SGTM